core/dbt/parser/read_files.py

```diff
@@ -94,9 +95,17 @@ def validate_yaml(file_path, dct):
 
 # Special processing for big seed files
 def load_seed_source_file(match: FilePath, project_name) -> SourceFile:
-    if match.seed_too_large():
+    if match.file_size() > MAXIMUM_SEED_SIZE:
         # We don't want to calculate a hash of this file. Use the path.
         source_file = SourceFile.big_seed(match)
+    elif match.file_size() <= DEFAULT_MAXIMUM_SEED_SIZE:
```

Comment:
The reason I wouldn't add utf-8 in general is because it is slower to make the conversion.  I'm not sure why there would be differences unless the files were of different sizes.  The only reason I thought of doing the utf-8 is so that when existing dbt users use this, they don't get hit with the seed file changed

Reply:
Alright let's keep this to reduce the unnecessary encoding and decoding steps and still not create state changes for existing users with smaller seed files.